DuckDB.NET.Data/DuckDBCommand.cs

```diff
@@ -21,38 +24,50 @@ public class DuckDbCommand : DbCommand
     public override bool DesignTimeVisible { get; set; }
     public override UpdateRowSource UpdatedRowSource { get; set; }
 
-    public override string CommandText { get; set; }
+    private string commandText = string.Empty;
 
-    protected override DbConnection DbConnection
+#if NET6_0_OR_GREATER
+    [AllowNull]
+#endif
+    [DefaultValue("")]
+    public override string CommandText
     {
-        get => connection;
-        set => connection = (DuckDBConnection)value;
+        get => commandText;
+        set
+        {
+            // TODO: We shouldn't be able to change the CommandText when the command is in execution (requires CommandState implementation)
+            commandText = value ?? string.Empty;
```

Comment:
What's `CommandState`?

Reply:
CommandState should manage the state of the command (Idle, InProgress, Disposed ...) to prevent any change (to commandText) when executing.